Cutter.cpp:3333:23: error: conversion from 'RzStrEnc' to 'QChar' is ambiguous

**Environment information**
* Operating System: MacOS 13.1 ARM64
* Cutter version: https://github.com/rizinorg/cutter/commit/7f0daf96ddcaf544fa8c2935afc04c2cc172abbf
* Obtained from:
  - [x] Built from source
  - [ ] Downloaded release from Cutter website or GitHub 
  - [ ] Distribution repository
* File format: -

**Describe the bug**
```
                                   ^
/Users/user/Rizin/cutter/src/core/Cutter.cpp:3333:23: error: conversion from 'RzStrEnc' to 'QChar' is ambiguous
        string.type = str->type;
                      ^~~~~~~~~
```

--- a/src/core/Cutter.cpp
+++ b/src/core/Cutter.cpp
@@ -3330,7 +3330,7 @@ QList<StringDescription> CutterCore::getAllStrings()
         StringDescription string;
         string.string = rz_str_escape_utf8_keep_printable(str->string, &opt);
         string.vaddr = obj ? rva(obj, str->paddr, str->vaddr, va) : str->paddr;
-        string.type = str->type;
+        string.type = rz_str_enc_as_string(str->type);
         string.size = str->size;
         string.length = str->length;
         string.section = section ? section->name : "";

